### ISSUE TITLE
index.js missing

### ISSUE THREAD

**Issue by ArthurClune** [regular/unbzip2-stream#16]
After installing 1.2.4 via npm on Ubuntu 14.04

```npm install unbzip2-stream```

 `require('unbzip2-stream')` gives  the error
```
Error: ENOENT: no such file or directory, open '/home/arthur/code/aws/node_modules/unbzip2-stream/index.js'
```
and indeed that file is missing. Grabbing a copy from the repo and putting it in place solves the issue, so it just appears to be missing from the package

```
arthur@biber:~/code/aws$ node --version v6.11.1
arthur@biber:~/code/aws$ npm --version   3.10.10                
```

**Comment by regular** [comment]
@ArthurClune can you check if it is fixed in 1.2.5? Thanks!

**Comment by ArthurClune** [comment]
Yes, that's fixed for me.

Thanks!
